src/app/story/directives/basic.component.ts

```diff
@@ -7,15 +7,15 @@ import { StoryModel, TabService } from '../../shared';
   template: `
     <form *ngIf="story">
 
-      <publish-fancy-field [model]="story" textinput="true" name="title" label="Story Title" required>
+      <publish-fancy-field [model]="story" textinput="true" name="title" label="Story Title" tiptext="This will show up in the \'Name\' column on iTunes." required>
```

Comment:
Alternatively, you could create a `class="fancy-tip"` element inside the field, the same way hints work.  And then pull that selector out in the component - which could also allow you to use html tags in the tooltip.  Okay, so maybe that's not all that useful.